rpm/user-managerd.spec

```diff
@@ -1,5 +1,5 @@
 Name: user-managerd
-Version: 0.0.1
+Version: 0.5.17+jb50231
```

Comment:
Version: 0.5.17 is enough

Reply:
I would say 0.5 is enough unless somebody is planning to maintain the spec version in sync with the reality.

Reply:
Version was updated because for some reason obs feature branch needed it to use proper version. Hopefully normal obs build uses the tag version as usual.

Reply:
Well, I do maintain those for my packages but it's totally up to the maintainer. If noone it going to sync those with tags then it's clearly better to have an obviously bugus value there, like 0.0.1